client/pages/user-list/list.template.html.slim

```diff
@@ -4,41 +4,57 @@
   tabset type='pills'
     tab ui-sref='admin.users' active='activeTabs.list'
       tab-heading User List
-      table.table.table-hover st-pipe='usersList.refresh' st-table='usersList.records'
-        thead
-          tr
-            th colspan=2 User
-            th.hidden-xs.hidden-sm.hidden-md E-mail
-            th.hidden-xs Registered since
-            th
-        tbody
-          tr ng-repeat='user in usersList.records'
-            td.avatar
-              a href='#' ui-sref='admin.users.show({ id: user.id })'
-                user-avatar user='user'
-            td.name
-              a ui-sref='admin.users.show({ id: user.id })' {{ user.name }}
-            td.email.hidden-xs.hidden-sm.hidden-md
-              span ng-if='user.primaryEmail' {{ user.primaryEmail }}
-              span ng-if='!user.primaryEmail'
-                | -
-            td.registeredAt.hidden-xs
-              | {{ user.createdAt | amDateFormat:'ll' }}
-              |  
-              | ({{ -timeFromNow(user.createdAt) | amDurationFormat:null:true }})
-            td.actions
-              .btn-group.btn-group-sm
-                a.btn.btn-danger ng-click='delete(user)' tooltip='Delete User' tooltip-append-to-body='true'
-                  span.glyphicon.glyphicon-remove
-        tfoot
-          tr
-            td.clearfix colspan=5 st-pagination=true st-items-by-page=15 st-template='/templates/services/tables/tables.pagination.template.html'
+      div st-pipe='usersList.refresh' st-table='usersList.records'
+        .well.filters
+          .row st-filters='usersList.params'
+            .col-md-4
+              organization-select prefix='users-list' placeholder='All organizations' model-object='usersList.params' only-active='false'
```

Comment:
The only-active attribute is not available in this branch but will be once the resque-stats branch will be merged to master and back there.